packages/@lwc/engine-server/package.json

```diff
@@ -46,6 +46,7 @@
         "@lwc/rollup-plugin": "3.6.0",
         "@lwc/shared": "3.6.0",
         "@rollup/plugin-virtual": "^3.0.1",
-        "parse5": "^7.1.2"
+        "parse5": "^7.1.2",
+        "@parse5/tools": "^0.3.0"
```

Comment:
is that basically because we'll bundle the dev dependencies and don't want consumers to also be installing them?

and shall i move parse5 while im in there?